modules/installation-configuration-parameters.adoc

```diff
@@ -575,11 +575,11 @@ ifndef::aws,bare,ibm-power,ibm-z,azure,ibm-power-vs[]
 |String
 endif::aws,bare,ibm-power,ibm-z,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure[]
+ifdef::aws,bare,azure,gcp[]
 |`compute.architecture`
```

Comment:
it seems double `compute.architecture` field in the doc [installation-config-parameters-gcp](https://62253--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp#installation-configuration-parameters-optional_installation-config-parameters-gcp)